CHANGELOG.md

```diff
@@ -197,11 +205,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Thanks to the people who made this release happen!
 
- * Martin von Zweigbergk (@martinvonz)
- * Danny Hooper (hooper@google.com)
- * Yuya Nishihara (@yuja)
  * Ilya Grigoriev (@ilyagr)
```

Comment:
Or by the number of commits to make people want to contribute more, both are fine with me. But the current order (first commit after the release) doesn't feel like a reasonable metric.

Reply:
I sorted by name. The old order was what it would have been if we had stuck to the scheme where every contributor adds their name. Then I think I just didn't think to question that order, so thanks for asking.